packages/react-icons/src/contexts/IconDirectionContext.ts

```diff
@@ -10,4 +10,4 @@ const IconDirectionContextDefaultValue: IconDirectionContextValue = {};
 
 export const IconDirectionContextProvider = IconDirectionContext.Provider;
 
-export const useIconContext = () => React.useContext(IconDirectionContext) ?? IconDirectionContextDefaultValue;
\ No newline at end of file
+export const useIconContext = () => React.useContext(IconDirectionContext) ? React.useContext(IconDirectionContext) : IconDirectionContextDefaultValue
```

Comment:
I'm not sure I understand why you're getting rid of the existing nullish coalescing operator `??`. It's functionally the same

Reply:
Ahh yes this is because of an issue where the nullish coalescing operator causes an error with some webpack configurations, so I just write it out